### ISSUE TITLE
OTP < 20 doesn't work on ubuntu 18.04 or 20.04

### ISSUE THREAD

**Issue by wojtekmach** [erlef/setup-beam#7]
Here's the workflow file: https://github.com/wojtekmach/setup-elixir-bug/blob/master/.github/workflows/ci.yml

and the results: https://github.com/wojtekmach/setup-elixir-bug/actions/runs/646018806

Everything is fine on ubuntu-16.04.

We're obviously about to have OTP 24 and the latest release in 19.x series was already 2 years ago (https://github.com/erlang/otp/releases/tag/OTP-19.3.6.13) so it's probably a low priority but thought I'd leave this as reference. Personally I wouldn't mind documenting this project to only support OTP 20+ and call it a day :)

**Comment by ericmj** [comment]
AFAIK 20.04 simply is not compatible with OTP < 20 so there is not much we can do. For 18.04 we can try changing the installed libssl version as we do here: https://github.com/hexpm/bob/blob/master/priv/scripts/docker/erlang-ubuntu-bionic.dockerfile#L44.

**Comment by starbelly** [comment]
I agree with @wojtekmach. Once OTP 24 is out I see no reason to support anything before OTP 20.

**Comment by ericmj** [comment]
I definitely want to keep supporting down to OTP 17 at least since that's what's required for Elixir 1.0. Dropping OTP 19 and older would mean also dropping Elixir 1.4 and older.

I would rather the compatible versions be documented than dropping support for a wide range of versions that currently work.

**Comment by starbelly** [comment]
@ericmj I see. Yeah, until elixir drops support for those, then agreed, it's a non-option to drop support for < 20

**Comment by paulo-ferraz-oliveira** [comment]
@ericmj, I agree with the version support range documentation (and not dropping for the sake of it). We should add it to `README.md` once #9 is merged, so we can also refer to `rebar3`. What do you think?

**Comment by ericmj** [comment]
Sounds good 👍

**Comment by paulo-ferraz-oliveira** [comment]
I created https://github.com/erlef/setup-elixir/issues/18, where @wojtekmach's issue is apparent.

**Comment by paulo-ferraz-oliveira** [comment]
Is there anything still we want to do within the scope of this Issue, or is it good to close?

**Comment by ericmj** [comment]
We need to document the incompatibility between OTP and Ubuntu versions and we should try to make ubuntu-18.04 work since it works on docker: https://github.com/erlef/setup-elixir/issues/7#issuecomment-797451084.

**Comment by paulo-ferraz-oliveira** [comment]
To be clear: "document the incompatibility between OTP and Ubuntu versions" means you want it in this (setup-elixir) repo.? or elsewhere? If it's in this repo. I can create an Issue to address later.

**Comment by ericmj** [comment]
Yes, I think it needs to be documented here because I can't find documentation for it anywhere else. For reference this is the incompatibility table we have for Bob's docker builds: https://github.com/hexpm/bob/blob/master/lib/bob/job/docker_checker.ex#L80-L83. If we fix the issue with ubuntu-18.04 then they should be the same.
